pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -454,8 +454,9 @@ func TestDesiredRouterDeploymentSpecAndNetwork(t *testing.T) {
 	var expectedReplicas int32 = 8
 	ic.Spec.Replicas = &expectedReplicas
 	ic.Spec.UnsupportedConfigOverrides = runtime.RawExtension{
-		Raw: []byte(`{"loadBalancingAlgorithm":"random","dynamicConfigManager":"false","maxConnections":-1,"reloadInterval":15}`),
+		Raw: []byte(`{"loadBalancingAlgorithm":"random","dynamicConfigManager":"false","reloadInterval":15}`),
 	}
+	ic.Spec.TuningOptions.MaxConnections = -1
```

Comment:
This could be moved above, here: https://github.com/openshift/cluster-ingress-operator/blob/6620e18087417fc5f0c922602826ce4e30ae1ed2/pkg/operator/controller/ingress/deployment_test.go#L430-L434

Reply:
Did you mean above? I added `MaxConnections` as an addition after `ThreadCount`.